app/models/case/base.rb

```diff
@@ -814,6 +814,7 @@ def overturned_ico?;         false;  end
   def overturned_ico_sar?;     false;  end
   def overturned_ico_foi?;     false;  end
   def offender_sar?;           false;  end
+  def offender_sar_standard?;  false;  end
```

Comment:
maybe give more background why a new type of offender sar needs to be introduced?